go.mod

```diff
@@ -263,3 +263,5 @@ require (
 	stash.kopano.io/kgol/kcc-go/v5 v5.0.1 // indirect
 	stash.kopano.io/kgol/oidc-go v0.3.2 // indirect
 )
+
+replace github.com/pkg/xattr => github.com/micbar/xattr v0.4.6-0.20220215082911-6552c78563ce
```

Comment:
add a commentd explaining why we use a fork? maybe link the PR?

Reply:
added